### DISCUSSION TITLE
Associate code block with a filename

### DISCUSSION THREAD

**Discussion by shivaraj-bh** [srid/emanote#471] [Ideas]
For example "flake.nix" [here](https://flakular.in/blog/nixify-haskell-project-intro#introduce-flake):
<img width="678" alt="Screenshot 2023-12-06 at 6 30 14 PM" src="https://github.com/srid/emanote/assets/23645788/684e3be9-ccc5-4307-bc56-a23741289b6e">

**Comment by srid** [comment]
We need a spec for this.
